src/main/java/io/vertx/core/net/impl/TCPServerBase.java

```diff
@@ -11,24 +11,13 @@
 package io.vertx.core.net.impl;
 
 import io.netty.bootstrap.ServerBootstrap;
-import io.netty.buffer.PooledByteBufAllocator;
-import io.netty.channel.Channel;
-import io.netty.channel.ChannelFuture;
-import io.netty.channel.ChannelFutureListener;
-import io.netty.channel.ChannelOption;
-import io.netty.channel.EventLoop;
-import io.netty.channel.EventLoopGroup;
+import io.netty.channel.*;
```

Comment:
pls no star imports :-)

Reply:
I'll update the PR.

This is a good example of the limitation of using "the IntelliJ defaults" as stated in https://github.com/vert-x3/wiki/wiki/Vert.x-code-style-guidelines